docs/app/Examples/addons/Portal/Types/PortalExampleControlled.js

```diff
@@ -23,12 +21,7 @@ export default class PortalExampleControlled extends Component {
             onClick={this.handleClick}
           />
 
-          <Portal
-            closeOnTriggerClick
-            openOnTriggerClick
-            onClose={this.handleClose}
-            open={open}
-          >
+          <Portal onClose={this.handleClose} open={open}>
```

Comment:
`closeOnTriggerClick` and `openOnTriggerClick` are useless for controlled example